frontend/server/src/main/java/com/amazonaws/ml/mms/wlm/WorkerThread.java

```diff
@@ -140,20 +142,24 @@ public void run() {
             }
         } catch (InterruptedException e) {
             logger.warn("Backend worker thread interrupted.");
-        } catch (WorkerInitializationException wi) {
-            logger.error("Backend worker error", wi);
+        } catch (WorkerInitializationException e) {
```

Comment:
Why this change?

Reply:
just make the exception handling more consistent.

Reply:
Makes sense :)